src/components/memberships/FetchMembershipsForm.js

```diff
@@ -73,8 +87,13 @@ class FetchMembershipsForm extends React.Component {
   }
 }
 
-export default connect(
-  state => ({
-    memberships: state.membeships,
-  })
-)(FetchMembershipsForm);
+function mapStateToProps(state, ownProps) {
+  const { user_email, team_name } = ownProps.match.params
+  return {
+    memberships: state.memberships,
+    user_email: user_email,
+    team_name: team_name
```

Comment:
You can check the issues with eslint. I think there should be a comma at the end